journal/original/03-chapter-2.adoc

```diff
@@ -92,7 +92,7 @@ a sweet harmony of love and life attended us,
 and we were comforted in the Lord and one another.
 We had good service at that meeting in helping Friends in discipline;
 and the meeting ended in the love of God.
-Tidings having reached us of the death of John ap John`'s wife,
+Tidings having reached us of the death of __John ap John`'s wife__,
```

Comment:
Same here, let's remove the italics around this phrase completely and leave it as it was.